PowerSimulationsPlots/src/plot_recipes.jl

```diff
@@ -3,6 +3,21 @@ import PlotlyJS
 import WebIO
 WebIO.install_jupyter_nbextension()
 
+function _format_multiple_plots(length::Int, plots::Array)
```

Comment:
Why is this function necessary?  Does `plots` have extra entries that you are truncating?  If so, could you write
```plots = plots[1:length]```

Reply:
couldn't figure this out for now....

Reply:
Add a comment on why the hacky workaround.